sites/gc-page-feedback/includes/gc-page-feedback.html

```diff
@@ -0,0 +1,62 @@
+{%- include_relative includes/parameters.liquid -%}
+{%- include_relative includes/strings.liquid -%}
+
+<div class="gc-opuft row">
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">{{ opuft_h3 }}</h3>
```

Comment:
This might produce a strange heading structure in the page details section since none of the other features that go in there (i.e. RAP/share this page/date modified) use headings. Especially since OPUFT precedes everything else.

That's why I was previously using ``figure``/``figcaption``; to structure OPUFT like a mini-section without making it as prominent as a real section (sorta like what RAP does with ``details``/``summary``). But @GormFrank wasn't very on it. Plus NVDA couldn't cope very well with a ``figure`` containing a ``form`` (IIRC it announced the figure... but skipped its ``figcaption`` and form mode announcements).